front/src/components/boxs/device-in-room/device-features/VolumeDeviceFeature.jsx

```diff
@@ -0,0 +1,69 @@
+import get from 'get-value';
+import { Text } from 'preact-i18n';
+import cx from 'classnames';
+
+import { DeviceFeatureCategoriesIcon } from '../../../../utils/consts';
+import { SIREN_VOLUME } from '../../../../../../server/utils/constants';
+
+const VolumeDeviceFeature = ({ children, ...props }) => {
```

Comment:
I'm not really sure that "low/medium/high" will be the only volume management for sirens.
I think we can have such 0 to 100% volume level.
Maybe we should setup a specific "LMH_VOLUME" feature?

Reply:
I'll see to create a new volume feature as your suggestion